jdeeco-core/src/cz/cuni/mff/d3s/deeco/runtimelog/RuntimeLogRecord.java

```diff
@@ -0,0 +1,69 @@
+package cz.cuni.mff.d3s.deeco.runtimelog;
+
+import java.util.Map;
+import cz.cuni.mff.d3s.deeco.model.runtime.api.ComponentInstance;
+
+/**
+ * This class defines a record that can be logged using the {@link RuntimeLogger}. 
+ * 
+ * @author Dominik Skoda <skoda@d3s.mff.cuni.cz>
+ */
+public class RuntimeLogRecord 
+{
+	/**
+	 * The type of the event being logged. There are no constraints on the types of event.
+	 */
+	private String eventType;
```

Comment:
I would get rid of the eventType. The eventType should be the actual type of the Record class. The RuntimeLogRecord itself should be abstract.